interface/orders/orders_results.php

```diff
@@ -392,25 +392,25 @@ function validate(f) {
 
  <tr class='head'>
   <td colspan='2'><?php echo $form_batch ? xl('Patient') : xl('Order'); ?></td>
-  <td colspan='4'><?php xl('Report', 'e'); ?></td>
-  <td colspan='7'><?php xl('Results and', 'e'); ?> <span class='reccolor''>
-    <?php  xl('Recommendations', 'e'); ?></span></td>
+  <td colspan='4'><?php xlt('Report', 'e'); ?></td>
+  <td colspan='7'><?php xlt('Results and', 'e'); ?> <span class='reccolor''>
+    <?php  xlt('Recommendations', 'e'); ?></span></td>
```

Comment:
For above 3 lines:
Correct use of xlt(), but need to remove the `e` parameter and add the `echo`